challenge.md

```diff
@@ -7,37 +7,37 @@ Write your answers inside this file, where it's indicated by the comments.
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
 <!-- Answer Starts Here -->
-
+git clone git@github.com:payne-chris-r/git-basics-08-assessment.git
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
 <!-- Answer Starts Here -->
-
+git status (git diff for more detail)
 <!-- Answer Ends Here -->
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
 <!-- Answer Starts Here -->
-
+If it's just the comment/message use git commit --amend. If you need to go back and redo the whole thing you can use git reset --hard, but that will also erase any changes you've made. I would likely just use git add and make a new commit including the changed files that were missed in the previous commit.
```

Comment:
Actually, no - `git commit --amend` can be used to redo anything about the last commit (including code), not just the commit message. In fact, although your approach might be feasible, by running `git reset --hard`, you'd actually be erasing any files that you'd changed since that commit - you'd need to redo all of those changes by hand.